Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:latest
```

Comment:
I've not run into problems with 5.X, but we're running FCC on 4.X so that's what contributors should be working from.